development/values.yaml

```diff
@@ -12,8 +12,12 @@ git:
   uri: https://github.com/rht-labs/open-management-portal-backend.git
   ref: master
 
-cacheService: false
```

Comment:
Should we also remove `CACHE_SERVICE` from `templates/secret.yaml` if it is no longer being used?

Reply:
forgot to commit a file.   it is now removed from the secret.yaml file as well.